.github/workflows/renovate.yml

```
@@ -0,0 +1,16 @@
+name: Renovate
+on:
+  schedule:
+    # Run every 30 minutes
+    - cron: "0/30 * * * 1"
```

```suggestion
    - cron: "0/30 23 * * 7"
    - cron: "0/30 0-5 * * 1"
```
Only run at night (with timezone correction UTC+2)